Nodejs/Product/TypeScriptSourceMapReader/TypeScriptSourceMapReader.csproj

```diff
@@ -0,0 +1,106 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="dogfood" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <ProductVersion>9.0.30729</ProductVersion>
+    <SchemaVersion>2.0</SchemaVersion>
+    <ProjectGuid>{61A8D7EF-B83E-4C21-9479-BA832357E42C}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>TypeScriptSourceMapReader</RootNamespace>
+    <AssemblyName>TypeScriptSourceMapReader</AssemblyName>
+    <FileAlignment>512</FileAlignment>
+    <UseVSVersion>true</UseVSVersion>
+    <GenerateAssemblyAttribute>false</GenerateAssemblyAttribute>
+    <ConvertLocalizableResourcesToDefaultCulture>True</ConvertLocalizableResourcesToDefaultCulture>
+    <UICulture>en</UICulture>
+    <GenerateLCE>true</GenerateLCE>
+    <SccProjectName>SAK</SccProjectName>
+    <SccLocalPath>SAK</SccLocalPath>
+    <SccAuxPath>SAK</SccAuxPath>
+    <SccProvider>SAK</SccProvider>
+    <OutputPath>$(BinariesDirectory)\bin\$(BuildArchitecture)\typescript\TypeScriptSourceMapReader</OutputPath>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>

```

Comment:
Just filed an issue here to address this later. https://github.com/Microsoft/nodejstools/issues/444